docs/Installation.md

```diff
@@ -0,0 +1,112 @@
+# Installation
+
+The AC Tool [content package](https://jackrabbit.apache.org/filevault) contains the tool itself and is available from [the Maven Central repository](https://repo1.maven.org/maven2/biz/netcentric/cq/tools/accesscontroltool/accesscontroltool-package/). 
+
+In addition, there needs to be another package containing the project/platform specific [AC Tool configuration](Configuration.md) which is typically part of the regular project source code and released along with it (e.g. as `myproject-permissions`).
+
+Due to the different nature of on-premise and cloud service infrastructure setup, the maven setup for installation of the two packages is slightly different (while the AC Tool configuration as such is compatible to both and can usually easily be lifted from on-premise to the cloud service).
+
+## Installation AEM On-Premise
+
+For AEM 6.4/6.5 use the following appraoch:
+
+* Use a complete package that contains all project-related packages as sub package (hence in `/etc/packages`)
+* Include the `accesscontroltool-package` *without* the `cloud` classifier (this has the [Sling Installer Provider Install Hook](https://sling.apache.org/documentation/bundles/installer-provider-installhook.html) in place)
+* Include the permissions package containing the project/platform specific [AC Tool configuration](Configuration.md)
+ * Ensure this permissions package declares the [dependency](#declaring-the-dependency) correctly
+
+
+Use the package with the GAV 
```

Comment:
There is no V below. Rather use "Maven coordinates" or say "group id and artifact id"